test/system.jl

```diff
@@ -279,6 +279,19 @@ ps = PathSet(; symlink=true)
                 compare_stats(stat(fp), stat(string(fp)))
                 compare_stats(lstat(fp), lstat(string(fp)))
             end
+
+            @testset "User/Group constructors" begin
```

Comment:
Before implementing the fix this set of tests was actually failing at `my_group = FilePathsBase.Group()` on the 32-bit machine, so outside the tests themselves.  But maybe the tests below serve as confirmation that the `Group` and `User` were successfully constructed?

Also, let me know if these tests would make more sense elsewhere.  I put them here because `User` and `Group` relate to `Status` which is tested in the block above.

Reply:
I might want to make a separate section for libc unit tests given that this is the second bug we've found, but this seems fine for now. Thanks for breaking up the reproducible error commit from the fix, that made this much easier to review 👍🏻